### PR TITLE
Add custom mouse wheel scrolling speed.

### PR DESCRIPTION
**Pull request by hugoeanogueira**
Two new properties were added `scrollingHorizontalMouseWheelMultiplier` and `scrollingVerticalMouseWheelMultiplier`, which are proxied to `finbars`.

These values can be set as:
- between 0 and 1 -> decrease speed;
- 1 -> original speed (default value);
- above 1 -> increase speed;
- negative values will invert the scrolling (could probably apply a modulo on this?...)

This should implement issue #290.
Thanks!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "chai": "^3.5.0",
     "extend-me": "^2.3",
     "fin-hypergrid-data-source-base": "^0.4.10",
-    "finbars": "1.5.2",
+    "finbars": "github:dashdash/finbars#FEATURE/custom_scroll_speed",
```
Comment (hugoeanogueira) [inline package.json]: This needs to be updated after approving the [pull request](https://github.com/openfin/finbars/pull/2) in `finbars` repository.

**Comment by Dwaynekj** [comment]
@hugoeanogueira @joneit And is this PR going stale?

**Comment by Dwaynekj** [comment]
@hugoeanogueira @joneit Are we ready to resume reviewing and resolving these prs?
as well as https://github.com/fin-hypergrid/core/pull/622
https://github.com/fin-hypergrid/core/pull/626

**Comment by joneit** [comment]
Closing. Worked into **Hypergrid v2.1.11** (PR #722).

**Pull request closed without merging**